atomix/cluster/src/main/java/io/atomix/cluster/messaging/ClusterCommunicationService.java

```diff
@@ -133,55 +133,44 @@
       boolean reliable);
 
   /**
-   * Sends a message and expects a reply.
+   * Multicasts a message to a set of members over TCP.
```

Comment:
This is not true right? It sends directly to the member with the id?

Reply:
Oops, the dangers of copy pasting :see_no_evil: